app/assets/javascripts/modules/reports.js

```diff
@@ -202,7 +203,8 @@ angular.module('probedock.reports', [ 'ngSanitize', 'probedock.api', 'probedock.
     };
   })
 
-  .controller('ReportDetailsCtrl', function(api, $scope) {
+  .controller('ReportDetailsCtrl', function(api, orgs, $scope) {
+    orgs.forwardData($scope);
```

Comment:
I assume this is superfluous now that the organization is passed to the directive as a scope parameter.